code/game/objects/items/devices/laserpointer.dm

```diff
@@ -126,12 +126,27 @@
 
 	//catpeople
 	for(var/mob/living/carbon/human/H in view(1,targloc))
-		if(ishumanbasic(H) && (H.getorgan(/obj/item/organ/tail/cat) || H.getorgan(/obj/item/organ/ears/cat) || H.dna.features["ears"] == "Cat" || H.dna.features["human_tail"] == "Cat"))
-			if(!H.incapacitated() && !H.lying)
-				H.visible_message("<span class='warning'>[H] pounces on the light!</span>","<span class='userdanger'>LIGHT!</span>")
-				H.Knockdown(10)
-				H.Move(targloc)
-				H.setDir(NORTH) //Facedown looks best imo
+		if(iscatperson(H)) //only real catpeople have the urge
+			if(!H.incapacitated() && !H.eye_blind)
+				if(!H.lying)
+					H.setDir(get_dir(H,targloc)) // kitty always looks at the light
+					if(prob(effectchance))

```

Comment:
this is the same var used for the borg stun right?

Reply:
and cameras. It should probbably also be the var used for humans, considering the human base chance is 33%, the same value as effectchance